tests/migrations/test_operations.py

```diff
@@ -381,6 +383,9 @@ def test_create_proxy_model(self):
             bases=("test_crprmo.Pony",),
         )
         self.assertEqual(operation.describe(), "Create proxy model ProxyPony")
+        self.assertEqual(
+            operation.formatted_description(), "+ Create proxy model ProxyPony"
+        )
```

Comment:
We don't need to add assertion for every operation in this file. It should be enough to cover every operation once. I will remove redundant assertion and double check that all operations are covered.